ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/misc.py

```diff
@@ -8,6 +8,9 @@
 
 from ansible_collections.arista.avd.plugins.filter.convert_dicts import convert_dicts
 from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
+from ansible_collections.arista.avd.plugins.filter.range_expand import range_expand
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_designs_facts.eos_designs_facts import EosDesignsFacts
```

Comment:
@ClausHolbechArista I think you need to update pyAVD build to reflect this.

Reply:
Actually nevermind, @hubert-arista just move this under the `if TYPE_CHECKING` block below and since `EosDesignsFacts` is part of the `eos_designs_facts` package you don't need to specify the module:
```
if TYPE_CHECKING:
    from ansible_collections.arista.avd.plugins.plugin_utils.eos_designs_facts import EosDesignsFacts
```